setup.cfg

```diff
@@ -15,7 +15,7 @@ ignore_missing_imports = True
 disallow_untyped_defs = True
 
 [tool:pytest]
-addopts = --flake8 --black --mypy --ignore-glob="**/test_*_instr.py"
+addopts = --flake8 --black --mypy --ignore-glob="**/test_*_instr.py" --ignore=doc/conf.py
```

Comment:
I suggest removing this change and modifying doc/conf.py (as shown below) bacause it allows pytest to run flake8 (and mypy) on the file, and it avoids the clumsy mypy exclusion system which requires listing each excluded file in two places (and the mypy section is a headache because mypy only allows a single value for `exclude` -- a single regex that has to match all the files you want to exclude).
```
from typing import TYPE_CHECKING
...
# This "if" works around a mypy problem with implicit namespaces.
# The symptom is:
# /opt/.../miniconda3-py38.../python/lsst/__init__.py:14:
#   error: Cannot determine type of "__path__"
if not TYPE_CHECKING:
    import lsst.ts.ess.csc  # noqa
```

If you are keen to do it here instead, please also add doc/conf.py to the [mypy] section (which is ugly, as described above).

Reply:
I adopted the TYPE_CHECKING if and reverted the change to setup.cfg.